test/mocha.opts

```diff
@@ -0,0 +1 @@
+../framework/test/mocha/mocha.opts
```

Comment:
Does it work with IDEs e.g. Webstorm? What to do when root test directory will have its own test files?

Reply:
Please checkout the branch and try it with your IDE. I don't have it installed. Also, I think IDE related configuration shouldn't be application configuration's concern.

When the root project have its own tests, they would conflict with the current framework tests anyway. I think we can come up with a solution then.

Reply:
I tried, its working with Webstorm.

Reply:
@yatki The point is not far to introduce root level tests for the application implementation. So start thinking of the solution to this problem.